apis/python/tests/test_dataframe.py

```diff
@@ -122,15 +122,15 @@ def test_dataframe_with_float_dim(tmp_path, arrow_schema):
     assert sdf.index_column_names == ("bar",)
 
 
-def test_dataframe_with_enumeration(tmp_path):
+@pytest.mark.parametrize("downgrade", [False, True])
```

Comment:
what is the `downgrade` param?

Reply:
oops went away on previous code-review-feedback param -- will remove